### PR TITLE
documented server.ts ( still need to add more ) + added dummyHPC

### PR DESCRIPTION
**Pull request by mitkotak**
create job with hpc == "dummy_hpc"

### CODE REVIEW

**Comment by mitkotak** [comment]
@zimo-xiao Can you have a look at this PR ?

File: server.ts [hunk 1]
```diff
@@ -789,6 +815,9 @@ app.post('/job/:jobId/submit', async function (req, res) {
             .where('id = :id', { id:  job.id })
             .set({ queuedAt: job.queuedAt })
             .execute()
+        if (job.hpc == "dummy_hpc") {
+            job.finishedAt = new Date()
```
Comment (zimo-xiao) [inline server.ts:819]: Could you please help explain why would we want to end the job before submit? My understanding is that this is for testing only.

File: configs/hpc.example.json [hunk 2]
```diff
@@ -79,5 +79,28 @@
         "init_sbatch_options": [
             "#SBATCH --partition=GPU"
         ]
+    },
+    "dummy_hpc": {
+        "ip": "bridges2.psc.edu",
```
Comment (zimo-xiao) [inline configs/hpc.example.json:84]: I would suggest you change the IP to the hpc's IP (in your case, the id of the docker container). Also, could you please add the key file as wekk as the docker-compose.yaml into the project? I could help you test the SSH functionality of the container.

**Review by zimo-xiao — CHANGES_REQUESTED** [review]

File: configs/hpc.example.json (hunk 2 above)
Comment (zimo-xiao) [reply configs/hpc.example.json:84]: My understanding was wrong. I though it was your dockerized HPC.

File: server.ts (hunk 1 above)
Comment (zimo-xiao) [reply server.ts:819]: My understanding was wrong. I though it was your dockerized HPC.

**Review by mitkotak — COMMENTED** [review]
Updated name + added sql query for job.finishedAt. Can you merge this commit as well

**Pull request merged**